src/index.js

```diff
@@ -1,14 +1,39 @@
-import _ from "lodash";
-import "./style.css";
+import './style.css';
+import Vert from '../icons/vert.png';
 
-function component() {
-  const element = document.createElement("div");
+const iconVert = new Image();
+iconVert.src = Vert;
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
-  element.classList.add("hello");
+const toDoTasks = [
+  {
+    description: 'wash the car',
+    bool: false,
+    index: 1,
+  },
+  {
+    description: 'buy food',
+    bool: false,
+    index: 2,
+  },
+  {
+    description: 'send email',
+    bool: false,
+    index: 3,
+  },
+];
```

Comment:
- Please, kindly use `completed` instead of `bool` to meet the project requirement.